DATA/SHIPS/shiparch.ini

```diff
@@ -1206,7 +1206,7 @@ DA_archetype = ships\utility\luxury_liner\luxury_liner.cmp
 material_library = ships\utility\utility_liner.mat
 material_library = fx\envmapbasic.mat
 envmap_material = envmapbasic
-linear_drag = 1
+linear_drag = 8751

```

Comment:
The 1 at the end can be just made to a 0, too. Looks more clean.

Reply:
Done